modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -22,6 +24,12 @@ def show
           submission = AppealsApi::EvidenceSubmission.find_by(guid: params[:id])
           raise Common::Exceptions::RecordNotFound, params[:id] unless submission
 
+          # rubocop:disable Style/IfUnlessModifier
```

Comment:
why disable this instead of just putting the if inline like im assuming its wanting?

Reply:
Yeah, that is a good question. I did it for two reasons... One is that it matches the shape of the other classes call sites and I like keeping things looking the same if possible. And the other reason is that I don't think having a `=`, `if`,  and`&&` all on one line is very readable...

```
submission = status_simulation_for(submission) if status_simulation_requested? && status_simulation_allowed?
```

Thoughts?